uint8 riser_distance # meters
uint16 riser_diameter # millimeter
uint8 delta_angle
uint8 delta_height
uint8 vertical_number
uint8 horizontal_number
bool use_rtk
---
bool result
